docs/aws/audit/sesmonitoring/rules/is_identity_verified.mdx

```diff
@@ -22,6 +22,222 @@ CBP
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all SES identities: Once your AWS CLI is set up, you can list all the Simple Email Service (SES) identities in your account by running the following command:
+   ```
+   aws ses list-identities
+   ```
+   This command will return a list of all identities (email addresses and domains) that you have attempted to verify.
+
+3. Check the verification status of each identity: For each identity listed, you can check its verification status by running the following command:
+   ```
+   aws ses get-identity-verification-attributes --identities "identity1" "identity2" "identity3"
+   ```
+   Replace "identity1", "identity2", "identity3" with the actual identities you want to check. This command will return the verification status for each identity.
+
+4. Analyze the output: The output of the previous command will show the verification status of each identity. If the verification status is "Pending", it means the identity is not verified. If the verification status is "Success", it means the identity is verified.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can do this by running the command `pip install boto3` in your terminal.
+
+2. Import the necessary modules and create a session using your AWS credentials. Here's a basic example:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'
+)
+```
+
+3. Now, you can use the `list_identities` method from the `ses` client to get a list of all identities (email addresses and domains). Here's how you can do this:
+
+```python
+ses_client = session.client('ses')
+
+response = ses_client.list_identities(
+    MaxItems=100
+)
+
+identities = response['Identities']
+```
+
+4. For each identity, you can use the `get_identity_verification_attributes` method to check if the identity is verified. If the `VerificationStatus` is not `Success`, then the identity is not verified.
+
+```python
+for identity in identities:
+    response = ses_client.get_identity_verification_attributes(
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>